src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlockBasic.java

```diff
@@ -334,3 +334,7 @@ void doNothingElse() { // ok
 
     }
 }
+
+interface InterfaceEndingWithSemiColon {
+    public void fooEmpty();
+};
```

Comment:
`// ok until #7541`

Reply:
please add this test case to general Inputs also.

Reply:
> please add this test case to general Inputs also.

Only `InputEmptyBlock.java` right? Others, catch block inputs.

Reply:
no, please add it to each UTs Inputs (`InputEmptyBlock.java` is ITs Inputs).
This case is such form is very spread code, our UT Inputs should have it. Input already have very similar code like:
```
interface Interface1
    {
        int i = 1;
        public void meth1(); };
```
but it is a bit weird, so on weird code weird behavior - it is hard to see that it is wrong.

Properly formatted code, with extra symbol is easy to understand.